mctp-estack/src/serial.rs

```diff
@@ -105,13 +81,13 @@ impl MctpSerialHandler {
                 Ok(1) => (),
                 Ok(0) => {
                     trace!("Serial EOF");
-                    return Err(Error::RxFailure);
+                    return Err(mctp::Error::RxFailure);
```

Comment:
Not sure if we need `mctp::`, most of the rest of the crate assumes `Error` and `Result` are the `mctp` implementations. Either way is OK though.

Reply:
I had removed the `use mctp::...` as we no longer use a bunch of `mctp` types (`Tag`, `MsgType`, etc). But it probably makes sense to keep `Result` and `Error` common, so I'll change to suit.

Reply:
done.